imports/plugins/core/discounts/server/no-meteor/register.js

```diff
@@ -0,0 +1,28 @@
+import startup from "./startup";
```

Comment:
Just an aside that if I was to nominate a plugin to be removed this would def be one of them